arches_for_science/media/js/views/components/workflows/project-report-workflow/download-report.js

```diff
@@ -69,7 +95,8 @@ define([
                 analysisAreas,
                 observationDetails: [...Object.values(observationDetails)],
                 projectDetails: [...Object.values(projectDetails)],
-                physicalThingsDetails: physicalThingsDetailsArray
+                physicalThingsDetails: physicalThingsDetailsArray,
+                objectOfStudyDetails: objectOfStudyDetailsArray
```

Comment:
Should the child objects (e.g. samples) appear in the report if I've selected the parent? When I select "Jefferson Memorial" as the object of interest, I also get its samples in the report:
<img width="641" alt="Introduction" src="https://github.com/archesproject/arches-for-science/assets/38668450/8c1c474a-fe7f-42db-aa4e-8bd300061dff">

Reply:
I believe thus is intentional since the sample is a real object unlike the areas.
But the requirements are still being refined, so it may be changed in the future, as well as the above (the request on collection) For example, do all the physical things to be included or just the one that the user selected...)
This should be good for now.